changelog/8881.removal.md

```diff
@@ -0,0 +1 @@
+Follow through on deprecation warnings and remove code, tests, and docs for `ConveRTTokenizer` and `LanguageModelTokenizer`.
```

Comment:
I think we'll need to add an entry to the migration docs as well?

Reply:
`ConveRTTokenizer`, `LanguageModelToknizer`, and `HFTransformerNLP` modules have a migration entry [here](https://rasa.com/docs/rasa/migration-guide#deprecations-2) along with what modules replace their functionality. Do you think we need to add this somewhere else?

Reply:
Yeah, the migration guide will need a new section for all removals in Rasa 3.0 (basically a sub-section under a section like [this](https://rasa.com/docs/rasa/migration-guide#rasa-110-to-rasa-20) ).. `2.8 to 3.0`. Removal of these components should be mentioned there.
You might want to check with enable on how what's the format of the subsection they are willing to follow for the migration guide.

Reply:
I don't think we can write the 2.8 -> 3.0 migration guide yet as we haven't finalized what 3.0 configs look like. Perhaps this should be a separate issue that I should make? We should probably also involve CSE.

Reply:
> I don't think we can write the 2.8 -> 3.0 migration guide yet as we haven't finalized what 3.0 configs look like.

How does that matter? 😅  We can anyways start a section to add all the migration bits right? Is there anything specific from the config that you'll need to add the migration entry for this?

> We should probably also involve CSE.

You can ask them if they have already thought of a format, but mostly they might have not because we are getting started with the revamp. Leaving it out for a separate issue seems risky to me.

Reply:
> How does that matter? 😅 We can anyways start a section to add all the migration bits right? Is there anything specific from the config that you'll need to add the migration entry for this? 

Just looking at the existing migration docs now they have before and after config examples and so I'm not sure what to put there now for the after sections since we haven't yet decided what after looks like.

My larger concern is that we haven't done this for the policy removals either but that PR already got merged so I think its just as risky to create a migration guide as part of this issue and risk leaving that stuff out. 

Would consider this PR ok to signoff on if I made an issue aggregating what needs to go into migration docs and put it in our inbox/point cse to it to start discussion?

Reply:
> Would consider this PR ok to sign off on if I made an issue aggregating what needs to go into migration docs and put it in our inbox/point cse to it to start discussion?

👍